### ISSUE TITLE
Refine the prompts

### ISSUE THREAD

**Issue by amchugh** [WilliamHCarter/RattlesnakeRidge#29]
Right now, the game works... but doesn't work too well. I'm unsatisfied with the quality. I think the best performance improvement can be found by improving the prompts given to chatgpt. If this is unsuccessful, maybe we need to select a stronger model.

**Comment by WilliamHCarter** [comment]
Agreed. We should make time to draft some strategies for better prompt engineering. I also want to ensure that the message history is being properly injected into the model every chat, and not just every conversation.
